test/unit-test/metric-middleware-koa-test.js

```diff
@@ -653,6 +653,57 @@ describe('metrics-middleware', () => {
                 Prometheus.register.clear();
             });
         });
+        describe('when using middleware request and path not matched to any specific route', () => {
```

Comment:
I had to spend a little time to understand how the tests were made and what level of things were being tested. So I believe this 1 unit test is sufficient for the code changes I've made. But please advise if there is anything else you request